src/OpenCV/Core/ArrayOps.hs

```diff
@@ -277,6 +310,23 @@ matScaleAdd src1 scale src2 = unsafeWrapException $ do
   where
     c'scale = realToFrac scale
 
+matMax
+    :: Mat shape channels depth -- ^
+    -> Mat shape channels depth
+    -> Mat shape channels depth
+matMax src1 src2 = unsafePerformIO $ do
```

Comment:
The result should be wrapped in a `CvExcept`. If the two input arrays are of a different size things will go wrong. This is not enforced by the type system. You can have `'D` for all type parameters.